modules/experimental/pages/daemonbane.adoc

```diff
@@ -0,0 +1,25 @@
+= Daemonbane
+
+`Detachment ability`
+
+These detachments specialise in fighting Psykers and Daemons.
+
+---
+
+[NOTE]
+====
+This ability is only for use when included in a specific scenario, as agreed by the players. It does not have a points cost and will have to be taken into account when writing the army lists for the scenario.
+====
+
+Where your detachment has the {daemonbane} ability:
+
+* In close combat, double the Assault value of each unit in the detachment that is in base contact with an enemy Daemon or {psyker} unit.
+* When you shoot with this detachment, apply a -1 modifier to the Armour value of each Daemon unit and each {psyker} unit in the target detachment.
+
+[NOTE]
+====
+Daemons include any unit in a Daemon detachment, and any Daemon Prince.
```

Comment:
A little wordsmithing, but also: Do you think that we should explicitly note that we exclude Daemon Engine units from the definition? It's just a thought — as its experimental anyway, we could skip over this for now if you prefer.

```suggestion
We define a Daemon unit as any of these units in a Chaos army: any unit in a Daemon Detachment; any Daemon Prince unit.
```